lib/action-creators.js

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Sure. As part of this patch (before we started MMMVP) I moved all our state into a single Redux 'store'. This means that we have no (or little) state living in the rest of our views or modules. Redux stores allow you to only change the application state via defined 'actions'. This file acts as an interface to the store that other modules can use to change app state.

Redux has been embraced by the React community as the best way to manage state. So much so that Facebook hired the author of Redux.